rest/src/main/groovy/whelk/rest/api/TransliterationAPI.groovy

```diff
@@ -0,0 +1,84 @@
+package whelk.rest.api
+
+import groovy.util.logging.Log4j2 as Log
+import whelk.util.Romanizer
+
+import javax.servlet.http.HttpServlet
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+import static whelk.util.Jackson.mapper
+
+@Log
+class TransliterationAPI extends HttpServlet {
+    private static final Set<String> supportedLangTags = Romanizer.romanizableLangTags()
+
+    @Override
+    void init() {
+        log.info("Starting Transliteration API")
+    }
+
+    @Override
+    void doPost(HttpServletRequest request, HttpServletResponse response) {
+        log.debug("Handling POST request for ${request.pathInfo}")
+
+        Map body = getRequestBody(request)
+
+        if (!(body.containsKey("langTag") && body.containsKey("source"))) {
+            log.warn("Transliteration parameter missing")
+            response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Parameter missing; needs langTag and source")
+            return
+        }
+
+        def languageTag = body["langTag"]
+        def source = body["source"]
+
+        if (supportedLangTags.contains(languageTag)) {
+            def romanized = Romanizer.romanize(source.toString(), languageTag.toString())
+            HttpTools.sendResponse(response,  romanized, "application/json")
+        } else {
+            log.warn("Language tag ${languageTag} not found")
+            response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Invalid language code")
+        }
+    }
+
+    @Override
+    void doGet(HttpServletRequest request, HttpServletResponse response) {
+        log.debug("Handling GET request for ${request.pathInfo}")
+
+        if (request.getPathInfo() == "/listLanguages") {
+            HttpTools.sendResponse(response, ["supportedLanguages": supportedLangTags], "application/json", HttpServletResponse.SC_OK)
```

Comment:
I think we reached the conclusion in the discussion that we don't need this?
Since get /language => 204/404 is enough.